nvtabular/workflow.py

```diff
@@ -869,7 +1014,7 @@ def build_and_process_graph(
 
         # Reorder tasks for two-phase workflows
         # TODO: Generalize this type of optimization
-        self.reorder_tasks()
+        #         self.reorder_tasks()
```

Comment:
Lets delete if we don't need this anymore (rather than just comment out).

Also can we delete the function definition itself? I believe this is the only place reorder_tasks is called